services/localfile/localfile.proto

```diff
@@ -267,3 +270,8 @@ message SymlinkRequest {
   string target = 1;
   string linkname = 2;
 }
+
+// MkdirRequest describes the directory
+message MkdirRequest {
+  FileWrite dir_description = 1;
```

Comment:
I'd mildly prefer using only FileAttributes

I don't think we should support Overwrite for MkdirRequest. It isn't something supported by `mkdir` and I'm not convinced that there's any behavior we could support for overwrite that isn't surprising.

Reply:
Oh, FileAttributes is initially the one I want, not sure why picked up FileWrite. Let me change it.